etcdctl/ctlv3/command/txn_command.go

```diff
@@ -115,6 +111,12 @@ func readOps(r *bufio.Reader) (ops []clientv3.Op) {
 	return ops
 }
 
+func promptInteractive(s string) {
```

Comment:
@heyitsanthony  sure will do this. My app was updating hence couldn't do it then.
Will fix.
Thanks!